library/src/commonMain/kotlin/org/jraf/klibnotion/internal/api/model/database/query/ApiDatabaseQuery.kt

```diff
@@ -25,13 +25,15 @@
 
 package org.jraf.klibnotion.internal.api.model.database.query
 
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 /**
  * See [Reference](https://developers.notion.com/reference/post-database-query).
  */
 @Serializable
 internal data class ApiDatabaseQuery(
-    val filter: ApiDatabaseQueryFilters? = null,
-    val sorts: List<ApiDatabaseQuerySort>? = null,
+    @SerialName("filter") val filter: ApiDatabaseQueryFilters? = null,
```

Comment:
I see how using `@SerialName("filter")` is probably cleaner, but I made the (possibly regrettable) choice early on to name the properties in Api models with the corresponding json name. For consistency, I would rather not add them in this one. 

(If you're motivated, maybe they could be all added in a separate PR? 😁)

Reply:
Haha, then sure I'll update them to lower_case, consistency is more important 😉